src/plugins/initiatives/discourse.test.js

```diff
@@ -0,0 +1,271 @@
+// @flow
+
+import {groupURLsByHeader, initiativeFromDiscourseTracker} from "./discourse";
+
+describe("plugins/initiatives/discourse", () => {
+  describe("initiativeFromDiscourseTracker", () => {
+    it("handles an example tracker", async () => {
+      // Given
+      const serverUrl = "https://foo.bar";
+      const topic = {
+        id: 123,
+        title: "Example initiative",
+        timestampMs: 1571498171951,
+        authorUsername: "TestUser",
+      };
+      const firstPost = {
+        id: 432,
+        topicId: 123,
+        indexWithinTopic: 1,
+        replyToPostIndex: null,
+        timestampMs: 1571498171951,
+        authorUsername: "TestUser",
+        cooked: `
+            <h1>Example initiative</h1>
+            <h2>Status: Testing</h2>
+            <h2>Champion<a href="https://foo.bar/t/dont-include/10"><sup>?</sup></a>:</h2>
+            <p>
+              <a class="mention" href="/u/ChampUser">@ChampUser</a>
```

Comment:
interesting. i believe as written, the discourse plugin expects the links to look like `https://discourse.sourcecred.io/u/s_ben`. i wonder if the discourse plugin is actually not detecting any mentions at all right now?

Reply:
Tracking that in https://github.com/sourcecred/sourcecred/issues/1420, I think this will be up to the discourse data source or reference detection. When that's updated this test can be changed to reflect it.